### DISCUSSION TITLE
unable to make contributions.

### DISCUSSION THREAD

**Discussion by Boby900** [community/community#42252] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

- I had switched the branches but while I am pushing the code to the Github it's showing **could not read from repository**

- I have forked and cloned the repo, added files and committed messages, switched the branches but still not working, your suggestions will be highly appreciated.

![Screenshot (12)](https://user-images.githubusercontent.com/111265239/208819195-3297869a-a3b3-4f99-97b1-a1207671f608.png)

**Comment by SilasRodrigues19** [comment]
Hi @Boby900 

If you are encountering this error message when trying to clone a Git repository using the `git clone` command, it means that either the repository does not exist or you do not have the necessary permissions to access it.

To resolve this issue, you can try the following steps:

1. Double-check the repository URL to make sure it is correct. You can do this by visiting the repository on the web and copying the URL from the address bar.

2. Make sure that you are using the correct Git protocol (e.g., `https` or `ssh`). Some repositories may require a specific protocol to be used.

3. If you are using the `https` protocol, try regenerating your personal access token and using that instead of your password.

4. If you are using the `ssh` protocol, make sure that you have set up your SSH keys correctly. You can test your SSH connection by running:  `ssh -T [git@github.com](mailto:git@github.com)`
 

5. If you are still having issues, you can try using the `git clone` command with the `--verbose` flag. This will provide more information about the clone process, which can help you troubleshoot the issue.

**Reply by Boby900** [reply]
ok, @SilasRodrigues19 thanks for your suggestions, I will follow these and will see if it's solving the problem.

**Reply by SilasRodrigues19** [reply]
@Boby900 Ok bro, good luck
